### ISSUE TITLE
Can Lockup Mortars

### ISSUE THREAD

**Issue by nmarshall23** [codetaylor/advancedmortars#13]
Steps to reproduce. 
1. Have a valid recipe in the Mortar
1. Place another item in. The Mortar's UI Changes showing you have invalid recipe. 
1. Remove that item,  The Mortar's UI Changes showing you have valid recipe.
1. Try to use the Mortar, it will not let you click until you remove all items, and place them back in. 

Here is the Recipe I am using.
```
Mortar.addRecipe(["diamond", "iron", "stone", "wood"], <minecraft:flint>, 12, [<minecraft:gravel>]);
```
I was testing the durations trying to get a feel of how long of a duration is a quick easy recipe and how long feels extensive. I would have no items in my hotbar, so when the flint was picked up I would click on the Mortar and place it in. That would result in an invalid recipe. Then I started troubleshooting. 

I suppose the right thing would be to not let an output of a recipe be added to the mortar if there is a valid Ingredients in it. That sounds like having to track state, not sure if it's worth it for a bug that easy to avoid. 

I checked my logs, no expectations were thrown.

**Comment by codetaylor** [comment]
Thanks for the report. This is a known issue and was fixed in #11. It will be in the next build later today.

**Comment by codetaylor** [comment]
This was happening because the recipe update was not being triggered on the server on item removal. When an item was removed on the server, changes would be sent to the client and that would trigger a recipe update on the client. At that point, the server would not see a valid recipe, while the client did. That's why the client was showing a valid recipe, but clicking did nothing.

Thanks again for the detailed report. :)
